src/languages/en.ts

```diff
@@ -1909,6 +1909,12 @@ export default {
         genericUpdateReportFieldFailureMessage: 'Unexpected error while updating the field, please try again later',
         genericUpdateReporNameEditFailureMessage: 'Unexpected error while renaming the report, please try again later',
         noActivityYet: 'No activity yet',
+        people: {
+            error: {
+                genericAdd: 'There was a problem adding this room member.',
+                genericRemove: 'There was a problem removing that room member.',
```

Comment:
This sounds a little bit of, especially `that room member` part. Did someone confirm this texts?

Reply:
I think what you have here is fine.

Reply:
Actually, let's add the `Waiting for copy` label and get official @Expensify/marketing sign-off.

Reply:
Ah, I don't think the "Waiting for copy" label works on pull requests! It looks like this is for a generic error message and we can't drill down into more specific reasoning?

Reply:
Also, where will these messages show up exactly and how will they look? 

Not knowing much else, I'd suggest slight tweaks:
- There was a problem adding this member to the room.
- There was a problem removing this member from the room.

Reply:
Getting a check [here](https://expensify.slack.com/archives/C21FRDWCV/p1708715288435469).

Reply:
This is already in the code it seems. I'm not an engineer, but an engineer confirmed it. @deetergp are you able to see the conversation [here](https://expensify.slack.com/archives/C21FRDWCV/p1708715288435469)? Thanks!